test/integration/targets/docker_host_facts/tasks/test_host_facts.yml

```diff
@@ -0,0 +1,235 @@
+---
+- name: Create random container name prefix
+  set_fact:
+    cname_prefix: "{{ 'ansible-test-%0x' % ((2**32) | random) }}"
+
+- debug:
+    msg: "Using container name prefix {{ cname_prefix }}"
+
+- name: Registering container name
+  set_fact:
+    cname: "{{ cname_prefix ~ '-hi' }}"
+
+- block:
+  - name: Get info on Docker host
+    docker_host_facts:
+    register: output
+
+  - name: assert reading docker host facts when docker is running
+    assert:
+      that:
+         - 'output.docker_host_facts.Name is string'
+         - 'output.docker_containers_list is not defined'
+         - 'output.docker_networks_list is not defined'
+         - 'output.docker_volumes_list is not defined'
+         - 'output.docker_images_list is not defined'
+         - 'output.docker_disk_usage is not defined'
+
+  - name: Create container
```

Comment:
I would add a comment that container and volume are created so that all lists are non-empty:
- container and volume lists are non-emtpy because of the created objects;
- image list is non-empty because the image of the container is there;
- network list is always non-empty (default networks).

Reply:
You mean as a comment lines at the playbook beginning or where exactly? Such comments are not part of other integration tests, usually, it is a rare situation to put them (not that I am against it)

Reply:
Added as a block

Reply:
Like you added it is fine. This is mostly a help for anyone else looking at the test, it makes it easier to understand why all queries return non-empty lists even though you only created a container and a volume.